docs/_docs/hivemq/schemas.md

```diff
@@ -0,0 +1,167 @@
+Validation API endpoint.
+
+```
+mqtt hivemq schemas
+```
+
+***
+
+## Commands
+
+| Command | Explanation                         |
+|---------|-------------------------------------|
+| get     | See [Get Schema](#get-schema)       |
+| delete  | See [Delete Schema](#delete-schema) |
+| create  | See [Create Schema](#create-schema) |
+
+***
+
+## Options
+
+### Help Options
+
+{% include options/help-options.md defaultHelp=true %}
+
+***
+
+# Get Schema
+
+*** 
+
+Retrieve a single existing schema.
+
+```
+mqtt hivemq schemas get
+```
+
+***
+
+## Simple Example
+
+```
+$ mqtt hivemq schemas get --id my-schema-id 
+
+{
+  "arguments": {},
+  "createdAt": "2023-05-25T21:10:42.779Z",
+  "id": "my-schema-id",
+  "schemaDefinition": "eyAidHlwZSI6ICJvYmplY3QiIH0K",
+  "type": "JSON"
+}
+```
+
+***
+
+## Options
+
+| Option | Long Version | Explanation                    | Default  |
+|--------|--------------|--------------------------------|----------|
+| `-i`   | `--id`       | The id of the schema to fetch. | required |
+
+### API Connection Options
+
+{% include options/api-connection-options.md %}
+
+### Logging Options
+
+{% include options/logging-options.md %}
+
+### Help Options
+
+{% include options/help-options.md defaultHelp=true %}
+
+***
+
+# Delete Schema
+
+*** 
+
+Delete a single existing schema.
+
+```
+mqtt hivemq schemas delete
+```
+
+***
+
+## Simple Example
+
+```
+$ mqtt hivemq schemas delete --id my-schema-id 
+```
+
+***
+
+## Options
+
+| Option | Long Version | Explanation                         | Default  |
+|--------|--------------|-------------------------------------|----------|
+| `-i`   | `--id`       | The id of the schema to be deleted. | required |
+
+### API Connection Options
+
+{% include options/api-connection-options.md %}
+
+### Logging Options
+
+{% include options/logging-options.md %}
+
+### Help Options
+
+{% include options/help-options.md defaultHelp=true %}
+
+***
+
+# Create Schema
+
+*** 
+
+Create a new schema and upload it to a HiveMQ node.
+The schema definition may be provided either directly from the command line or from a file.
+
+```
+mqtt hivemq schemas create
+```
+
+***
+
+## Simple Example
+
+```
+$ mqtt hivemq schemas create --id my-schema-id --type protobuf --file my-schema.desc --message-type Message 
+```
+
+***
+
+## Options
+
+| Option | Long Version      | Explanation                                                                                                                                 | Default                                    |
+|--------|-------------------|---------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------|
+| `-i`   | `--id`            | The id of the schema to be created.                                                                                                         | required                                   |
```

Comment:
It really feels like we should add a Required column in some cases, but I would not include the information in the Default column.

Reply:
Would you also include additional notes in this Required column for `--file` and `--definition`, which are mutually exclusive, and `--message-type` and `--allow-unknown`, which are dependent on `--type protobuf`?

Reply:
Sure, sounds good. If you think the additional information fits better inside the Explanation column you can also write it there. I think both ways are equally good.